app/views/bounties/_bounty.html.erb

```diff
@@ -1,4 +1,4 @@
-<div id="<%= dom_id bounty %>" class="mb-8 bg-white border shadow rounded-md pt-8 px-8">
+<div id="<%= dom_id bounty %>" class="mb-8 bg-white border shadow rounded-md pt-8 px-8 [overflow-wrap:anywhere]">
```

Comment:
added `overflow-wrap:anywhere` on entire card tile so that if anything new (new information) added on tile, then it will be taken care of